commonknowledge/stopwatch
# Implement pinning articles

## Description
This PR adds a new block to the category page which allows the editor to select a number of posts/pages to pin to the top the page.
A desired style whether grid or rows can also be selected
This functions separately to the sort and filter functions on the page

## Motivation and Context
Addresses [STW-49](https://linear.app/commonknowledge/issue/STW-49/customise-grids-and-rows)

## How Can It Be Tested?
Download the branch and run locally
Run `python manage.py setup_pages --scratch True --ensure-site True --ensure-pages True` to set up some category pages. Create some article pages as child pages.
Edit a category page and add a pinned pages block and select a style

![Untitled1](https://github.com/commonknowledge/stopwatch/assets/43313455/8177fa9a-78f4-4519-b432-3ffedd1fec55)
View on page
![1reg](https://github.com/commonknowledge/stopwatch/assets/43313455/e0f18f52-b020-46a5-9179-2f472a193073)


## How Will This Be Deployed?
Normal deploy to Render

## Screenshots (if appropriate):

## Types of changes
